### ISSUE TITLE
CSS issues in RTD theme

### ISSUE THREAD

**Issue by fedelibre** [mkdocs/mkdocs#287]
As you can see [here](http://mkdocs.readthedocs.org/en/latest/about/license/), the font size of a list element is smaller than the paragraph.

font-size of ``<p>`` is 16 px while the font-size of ``<li>`` is 90% (inherited by body)

If you find other similar issues please report them here.

**Comment by d0ugal** [comment]
This was fixed in #394

**Comment by fedelibre** [comment]
@d0ugal I stumbled on this again on version 0.14.0
The workaround is using a line break after each list item, i.e. instead of this:

```md
- this is
- a list
- without breaks
```

I use this:

```md
- this is a list

- with

- breaks
```
